src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -356,6 +374,21 @@ private void deleteStudyCreationRequest(String studyName, String userId) {
         return webClient.post()
                 .uri(networkConversionServerBaseUri + path)
                 .retrieve()
+                .onStatus(httpStatus -> httpStatus == HttpStatus.INTERNAL_SERVER_ERROR, clientResponse ->
+                        clientResponse.bodyToMono(String.class).flatMap(body -> {
+                            try {
+                                String message;
+                                JsonNode node = new ObjectMapper().readTree(body).path("message");
```

Comment:
You probably could share code with importCase?

Reply:
Done